src/index.js

```diff
@@ -119,7 +132,7 @@ function setFullPaths(paths, key = '') {
   return Object.values(paths || []).map((module) => path.join(basePath, key ? module[key] : module));
 }
 
-const hyvaModuleDirs = hyvaThemesConfig && setFullPaths(hyvaThemesConfig.extensions, 'src');
+const hyvaModuleDirs = hyvaThemesConfig ? setFullPaths(hyvaThemesConfig.extensions, 'src') : [];
```

Comment:
Thats a no can do, `hyvaThemesConfig` or the `hyvaModuleDirs` needs to be returned as a valid value instead of `false`.

If this was `false`, the foreach loop (on line 140) will trow an error, so this needs to return a array, even if empty.

The `setFullPaths` did this in my previous commit, if the `hyvaThemesConfig` was always returned as an object.

Seeing I had to revert this to a `object` or `false`, the value the `hyvaModuleDirs` needed to be updated to handle the possible `false` value from `hyvaThemesConfig`.

And stated in the export comment for `hyvaModuleDirs`.
The return value should be an array and a false would not be the expected value.

```
  // Array of absolute paths to Hyvä modules
  hyvaModuleDirs,
```